dataverse-webapp/src/test/java/edu/harvard/iq/dataverse/dataaccess/FileAccessIOTest.java

```diff
@@ -41,11 +43,20 @@ public class FileAccessIOTest {
     private byte[] datafileBytes = "datafile content".getBytes();
 
     private byte[] dataToBeSaved = "tobesaved".getBytes();
-    private File fileToBeSaved = new File("/tmp/files/fileToBeSaved");
-
+    
+    private String tempFiles = "/tmp/files";
+    private File fileToBeSaved;
+    
 
     @Before
     public void setUpClass() throws IOException {
+    	
+    	if(System.getProperty("os.name").toLowerCase().contains("win")) {
+    		tempFiles = "c:/tmp/files";
```

Comment:
There's a `TemporaryFolder` rule for junit4, which can be used to handle temporary files during text execution. There's also an equivalent `@TempDir` for junit5, but I will handle that in my PR https://github.com/CeON/dataverse/pull/2561 which migrates all the tests to junit5.